scripts/start_emulator.sh

```
@@ -3,16 +3,36 @@
+echo "✅ Docker daemon is running"
+
 # Get the script directory
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 PROJECT_DIR="$(dirname "$SCRIPT_DIR")"
 
 # Create data directory if it doesn't exist
 mkdir -p "$PROJECT_DIR/docker/dynamodb"
 
-# Start DynamoDB Local using Docker Compose
+# Check if DynamoDB container is already running
 cd "$SCRIPT_DIR"
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)